apps/nextjs-website/react-components/components/Editorial/Ctas.tsx

```diff
@@ -20,9 +19,10 @@ export const Ctas = ({
           '@media screen and (min-width: 420px)': {
             flexDirection: 'row-reverse',
           },
+          rowGap: 2,
+          columnGap: 2,
```

Comment:
```suggestion
          gap: 2,
```
Would this not work?

Also, any idea why spacing parameter didn't work?

Reply:
spacing worked only for the same direction of the flex (in this case, column) so it regulated only the vertical gap. I separated only 'cause I don't know which distance we need for the x, otherwise we could merge into "gap" instead of twice gap

Reply:
The value of `flexDirection` changes depending on the screen's width so that shouldn't be the problem.

I looked into it and, I don't know why, but `spacing` doesn't set the `gap` property, it sets `marginLeft` (or `marginTop` if stacking vertically) for the children of the element.

Basically spacing isn't working because we're setting a value of 0 for the storeButtons margins.
Removing said values should make `spacing` work again.